services/storage/src/simcore_service_storage/dsm.py

```diff
@@ -963,10 +979,46 @@ async def search_files_starting_with(
             )
 
             async for row in conn.execute(stmt):
-                meta = FileMetaData(**dict(row))
-                meta_extended = FileMetaDataEx(
-                    fmd=meta,
-                    parent_id=str(Path(meta.object_name).parent),
-                )
+                meta_extended = to_meta_extended(row)
                 files_meta.append(meta_extended)
+
         return list(files_meta)
+
+    async def create_soft_link(
+        self, user_id: int, target_uuid: str, link_uuid: str
+    ) -> FileMetaDataEx:
```

Comment:
Is the file's `etag` taken int account here? Could this be an issue? If no, why?

Reply:
I am not sure i understand the impact of the ``etag`` here. 
The idea is that every file uploaded to s3 has a ``file_uuid`` associated to a ``object_uuid`` (i.e. the identifier in s3). Up to now, both identifiers were the same but now we can create a "solft-link" with a different ``file_uuid`` while preserving the ``object_uuid``. 
therefore there is no issue with the ``etag``